byteparse/shared/src/test/scala/byteparse/classparse/ClassTests.scala

```diff
@@ -0,0 +1,1180 @@
+          "ENh4FNh+yAAK7ACZZtwAnEi62ACkVHhUfYLYAKrYAK7YABLIAArsAJlm3ACcSL7YAKRUfFR5ktg" +
+          "AqtgArtgAEsgACuwAmWbcAJxIwtgApFR8VHmi2ACq2ACu2AASyAAK7ACZZtwAnEjG2ACkVHhUfb" +
+          "LYAKrYAK7YABLIAArsAJlm3ACcSMbYAKRUehxUfh2+2ADK2ACu2AASyAAISM7YABLIAAhI0tgAE" +
+          "sgACEjW2AASyAAISNrYABLIAAhI3tgAEsgACEji2AASyAAISObYABLIAAhI6tgAEsgACEju2AAS" +
+          "yAAISPLYABAM2ILIAAhI9tgAEsgACFSCEIAG2AD6yAAKEIAEVILYAPrIAAhUghCD/tgA+sgAChC" +
+          "D/FSC2AD6yAAISP7YABBAKNiEVIRAKoAAOsgACEkC2AASnAB0VIRAKpAAOsgACEkG2AASnAAuyA" +
+          "AISQrYABAM2IhUiEGSiABGyAAIVIrYAPoQiAaf/7rIAArsAJlm3ACcSQ7YAKRUitgAqtgArtgAE" +
+          "AzYjsgACFSO2AD6EIwEVIxBkof/xsgACuwAmWbcAJxJEtgApFSO2ACq2ACu2AAQDNiQVJBAKogA" +
+          "RsgACFSS2AD6EJAGn/+4DNiQVJBAKogAoAzYlFSUQCqIAGBUkCKAADBUlCKAABqcAD4QlAaf/54" +
+          "QkAaf/1xAJvApZAwRPWQQFT1kFBk9ZBgdPWQcIT1kIEAZPWRAGEAdPWRAHEAhPWRAIEAlPOiQZJ" +
+          "DolGSW+NiYDNicVJxUmogAYGSUVJy42KLIAAhUotgA+hCcBp//nBjYlFSWqAAAALgAAAAEAAAAD" +
+          "AAAAGQAAACAAAAAnEkU6JqcAFRJGOianAA4SRzompwAHEkg6JrIAArsAJlm3ACcSSbYAKRkmtgA" +
+          "ptgArtgAEEko6JxknOigCNikZKLYAS6sAAABOAAAAAwAADcEAAAAxAAHS5wAAACEGLexoAAAAQR" +
+          "koEky2AE2ZACYDNimnACAZKBJOtgBNmQAWBDYppwAQGSgSSrYATZkABgU2KRUpqgAAADoAAAAAA" +
+          "AAAAgAAABkAAAAkAAAAL7IAAhJPtgAEpwAzsgACElC2AASnACiyAAISUbYABKcAHbIAArsAJlm3" +
+          "ACcSUrYAKRkntgAptgArtgAECDYoFSgQCqIACBJTpwAFElQ6KbIAAhkptgAEElW4AFZXEHu4AFd" +
+          "XsQAAAAIAXAAAAb4AbwAAAAgACAAJABAACgAYAAsAIAAMADYADwA5ABEAPwATAEMAFABIABUATQ" +
+          "AWAFEAFwBWABgAWQAZAFwAGgBgAB0AZQAeAHAAHwB9ACAAiAAhAIwAIgCQACMAlAAkAJwAJQCkA" +
+          "CYArAAnALIAKAC4ACkAvgAqANUAKwDvACwBAAAtARwALgEhAC8BPQAwAUsAMQFoADIBhQAzAaIA" +
+          "NAG/ADUB3gA2AeYANwHuADgB9gA5Af4AOgIGADsCDgA8AhYAPQIeAD4CJgA/Ai4AQAIxAEECOQB" +
+          "CAkQAQwJPAEQCWgBFAmUARgJtAEcCcQBIAngASQKDAEoCigBLApUATQKdAE8CoABQAqcAUQKvAF" +
+          "ICtQBUAs8AVQLSAFcC2gBYAt0AWQLkAFoC/gBbAwgAXAMQAFsDFgBfAyAAYAMqAGEDNgBiAzkAY" +
+          "AM/AF8DRQBmA3YAZwOQAGgDmABnA54AagOhAGwDvABtA8AAbgPDAG8DxwBwA8oAcQPOAHID0QBz" +
+          "A9UAdgPvAHcD8wB4BGgAegRwAHsEcwB9BHsAfgR+AIAEhgCBBIkAgwSjAIYEpgCHBLYAiAS+AIk" +
+          "ExACKBMoAiwBeAAAA2QAf/wKDAB8HAF8BAQEBAQEEAgMBAQEAAwcAYAcAYQcAYQcAYgcAYgcAYg" +
+          "cAYwcAZAcAZQcAYwcAZAcAZQEBAQEAABEH/AACART8ABwB/AAuAfoAFPwAAgH8AAkBFfoABfoAB" +
+          "f8APAAlBwBfAQEBAQEBBAIDAQEBAAMHAGAHAGEHAGEHAGIHAGIHAGIHAGMHAGQHAGUHAGMHAGQH" +
+          "AGUBAQEBAQEHAGMHAGMBAQAA+AAb/AAdAQYGBvwAAwcAYv4ASgcAYgcAYgEPDwwaCgoK+QAZ/AA" +
+          "OAUEHAGIAAQBmAAAAAgBn").toByteArray
+        /* CodeTest.class from compiled class CodeTest.java */
+
+
+        val Parsed.Success(parsedClassInfo, _) = ClassParser.classFile.parse(classFile)
+        val parsedClass = ClassParser.Ast.convertToAst(parsedClassInfo)
+
+        assert(parsedClass.methods(1).attributes(0).asInstanceOf[CodeAttribute].code ==
+            ArrayBuffer(
```

Comment:
Question: we are asserting that this literal is the output of `.code`, how do we know that this expected output is correct? Did you compare it against some other tool e.g. `javap` to validate that all the bytecodes are the same, and all the indices line up?